Gemfile

```diff
@@ -5,3 +5,4 @@ gemspec
 
 gem "rake", "~> 12.0"
 gem "rspec", "~> 3.0"
+gem 'html2asciimath', git: 'git@github.com:plurimath/html2asciimath.git'
```

Comment:
@ronaldtse We need to publish this gem.
Let me know if I need to get it done?

Reply:
This is now published. Thanks!

Reply:
This has been just published now.